src/types/User.ts

```diff
@@ -1,6 +1,36 @@
+// export interface User {
+//   id: number;
+//   name: string;
+//   email: string;
+//   phone: string;
+// }
+
+interface Geo {
+  lat: string;
+  lng: string;
+}
+
+interface Address {
+  street: string;
+  suite: string;
+  city: string;
+  zipcode: string;
+  geo: Geo;
+}
+
+interface Company {
+  name: string;
+  catchPhrase: string;
+  bs: string;
+}
```

Comment:
Unused & useless

Reply:
We created this user type in a previous lesson so i just decided copy instead of user type which was by default